monkey/monkey_island/cc/services/config_schema/monkey.py

```diff
@@ -20,9 +20,9 @@
                     "title": "Linux post breach file",
                     "type": "string",
                     "format": "data-url",
-                    "description": "File to be executed after breaching. "
-                                   "If you want custom execution behavior, "
-                                   "specify it in 'Linux post breach command' field. "
+                    "description": "File to be uploaded after breaching. "
+                                   "If you want the file to be executed, "
+                                   "specify it in the 'Linux post breach command' field. "
```

Comment:
```suggestion
                    "description": "File will be uploaded after breaching. "
                                           "Use 'Linux post breach command' field to "
                                           "change permissions, run or delete the file."
```

Reply:
"File will be uploaded" doesn't seem right as a description. "File to be uploaded" is what I would expect.